python/paddle/fluid/contrib/slim/quantization/quantization_mkldnn_pass.py

```diff
@@ -0,0 +1,227 @@
+#   Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+from .... import core
+from ....framework import IrGraph
+from ....framework import IrNode
+
+__all__ = ['TransformForMkldnnPass']
+
+
+class TransformForMkldnnPass(object):
+    def __init__(self, scope=None, place=None):
+        """
```

Comment:
These comments are not for `__init__` function, they are for `class TransformForMkldnnPass`. Thus, move them after line 23?

Reply:
Agree. Thanks

Reply:
Done